### ISSUE TITLE
Add an option to change the highlighting background color of `updatemenus`

### ISSUE THREAD

**Issue by salim-b** [plotly/plotly.js#1428]
Currently only the normal (i. e. unhighlighted) background color can be changed through the parameter [`bgcolor`](https://plot.ly/javascript/reference/#layout-updatemenus-bgcolor). If you set this background to a dark coloring you most probaby also want to [adjust the text font](https://plot.ly/javascript/reference/#layout-updatemenus-font-color) to a bright coloring so that the text is still readable. But then the text becomes pretty unreadable while being highlighted since the highlighting background color is very bright!

Therefore I think **an additional parameter `bgcolor_active` or the like should be introduced.**

Additionally an even more sophisticated solution could involve an automatic adjustment of the text font color and/or background color depending on the color brightness of the user defined `bgcolor` and/or `font.color` values (this seems to be already the case with the font color of hoverinfo labels).

**Comment by jsandiego** [comment]
I'd like to bump this issue.  Would be great to have this customization!

**Comment by vidasci** [comment]
I would also love to see this being implemented, plotly plots with a dark theme look great but it's a big downside having a bright background in updatemenus

**Comment by alexandre-pecorilla** [comment]
Bump!
Like @vidasci said, the plotly_dark them is awesome but the bright background on the dropdown makes it impossible to see the values when hovering.

**Comment by Kinchul** [comment]
Bumping again ! It would be very nice to be able to customize these buttons colors !

**Comment by isaacbshelton** [comment]
@Kinchul I found a workaround in python that may be helpful. See here:
https://github.com/plotly/plotly.py/issues/3120#issuecomment-1018308604

**Comment by henrychoy** [comment]
I'm struggling to find a workaround in js.  Can anyone help?

**Comment by guidoTegaSistemas** [comment]
> I'm struggling to find a workaround in js. Can anyone help?
Not a full fix, but that's the best I got for today:
plotLayout['modebar'] = { color: 'white', activecolor: 'lightgray',bgcolor: 'rgba(100, 100, 100, 0.7)' };

**Comment by guidoTegaSistemas** [comment]
You are correct, I was having a similiar issue but I guess it won't work for this, in any case I believe those are the variables you could tinker with:
https://plotly.com/python/reference/layout/updatemenus/

**Comment by lopezvoliver** [comment]
Exposing an "activebgcolor" for `updatemenus` would be great 

even the AI in plotly's documentation (https://plotly.com/python/reference/layout/updatemenus/#layout-updatemenus) suggested it (but it doesn't exist yet):

> To change the background color of the dropdown menu when it is hovered, you can use the activebackgroundcolor property within the dropdown attribute in the update_layout method. By setting the activebackgroundcolor property to a specific color, you can customize the background color of the dropdown menu when it is hovered.

For now, the workaround that worked for me (within a [solara](solara.dev) app) is to add this in my `custom.css`:

```css
.theme--dark .updatemenu-item-rect {
    fill: #506784 !important;   
    fill-opacity: 1 !important; 
    stroke: #506784 !important; 
}

.theme--dark .updatemenu-item-rect:hover {
    fill: #ff8800 !important;       
    fill-opacity: 1 !important;     
    stroke: #ff8800 !important;     
}

.theme--dark .updatemenu-item-text {
    pointer-events: none;
}
```

You could also adjust the colors for the light theme using `.theme--light`

For context (this is solara-specific), I am using 

```python
    dark_effective = solara.lab.use_dark_effective()
```

in my component, and for plotly I assign the theme like this:

```python
 fig.update_layout(
    template='plotly_dark' if dark_effective else 'plotly_white',
    # other options here...
)
```
